bgpd/bgp_vty.c

```diff
@@ -199,7 +199,24 @@ static enum node_type bgp_node_type(afi_t afi, safi_t safi)
 	case AFI_L2VPN:
 		return BGP_EVPN_NODE;
 	case AFI_LINKSTATE:
-		/* TODO */
+		switch (safi) {
+		case SAFI_LINKSTATE:
+			return BGP_LS_NODE;
+		case SAFI_LINKSTATE_VPN:
+			/* Not yet supported */
+			return BGP_IPV4_NODE;
```

Comment:
no need to return BGP_IPV4_NODE. go to fallback.

Reply:
done